.vscode/settings.json

```diff
@@ -0,0 +1,48 @@
+{
+  "explorer.fileNesting.enabled": true,
+  "explorer.fileNesting.expand": false,
+  "explorer.fileNesting.patterns": {
+    ".prettierrc.json": ".*, *.md, LICENSE"
```

Comment:
This makes no sense.

Reply:
Why? This groups multiple files that are rarely edited by contributors.